src/ARCTokenization/TopLevelParsers.fs

```diff
@@ -9,54 +9,68 @@ module internal ISA =
 
     open System.IO
 
-    let tryParseMetadataSheetFromToken (isaFileName: string) (isaMdsParsingF: string -> IParam list) (absFileToken: IParam) =
+    let tryParseMetadataSheetFromToken (rootPath:string) (isaCvTerm: CvTerm) (isaMdsParsingF: string -> IParam list) (absFileToken: IParam) =
```

Comment:
nitpick: rename absFileToken -> relFileToken, since we ar enow tokenizing the farc file system as relative paths by convention